### PR TITLE
Fix layout when clause that assumed computes exist

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: roles/reproducer/tasks/libvirt_layout.yml [hunk 1]
```diff
@@ -91,7 +91,9 @@
 # match the multi-hypervisor schema.
 - name: Configure compute network
   when:
-    - "compute.key in groups['computes'] or compute.key in (groups['cephs'] | default([]))"
+    - >-
```
[the remark this answers is not in the document]
Comment (pablintino) [reply roles/reproducer/tasks/libvirt_layout.yml:94]: We have multiple examples in the framework using that notation. Why is this one an issue?
(Check `- >-` usages)
In this case, we can simplify it and directly do like this:
```yaml
- name: Configure compute network
  when: >-
     (compute.key in (groups['computes'] | default([]))) or
     (compute.key in (groups['cephs'] | default([])))
```

**Comment by eshulman2** [comment]
tested in my environment the fix is working

File: roles/reproducer/tasks/libvirt_layout.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (cjeanner) [reply roles/reproducer/tasks/libvirt_layout.yml:94]: oh, well, just not used to get string in there - but seems to be used in some other places (not THAT widely though ;) ).

**Review by rebtoor — APPROVED** [review]

**Pull request merged**